miniserver/rpc.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios')
+const config = require('./config')
+const { ethers } = require('ethers')
+const utils = require('../server/utils')
+const { hexStringToBytes } = utils
+
+const rpc = {
```

Comment:
return BN or string-form. BN is more general than ethers.js and used in many cryptography libraries

Reply:
also this can be just using server's

Reply:
**Background**
This code was copied from [one-wallet](https://github.com/polymorpher/one-wallet/blob/a0c737d9b42c2208ac5285f90a33dbfd853975f9/code/relayer/rpc.js#L19) which also returns  `return bn.toNumber()`
Happy to change it
Server does not have a `rpc.js` or a `getNonce`

Also I have been using [ethers BigNumber](https://docs.ethers.io/v5/api/utils/bignumber/#BigNumber--BigNumber--notes--why-not-bignumber-js-bn-js-bigdecimal-etc) which internally uses [BN.js](https://docs.ethers.io/v5/api/utils/bignumber/#BigNumber--BigNumber--notes--why-not-bignumber-js-bn-js-bigdecimal-etc). There reasoning for using this seems to make sense 
[Why not BigNumber.js, BN.js, BigDecimal, etc?](https://docs.ethers.io/v5/api/utils/bignumber/#BigNumber--BigNumber--notes--why-not-bignumber-js-bn-js-bigdecimal-etc)

Finally currently we have both ethers and web3 used throughout the project.
As mentioned [here3](https://github.com/polymorpher/sms-wallet/issues/13#issuecomment-1239889581)
I'd like to consistently use ethers rather than web3, should be able to consolidate code codebase to use ethers in a couple of hours.

Having said all that Happy to go with whatever approach we feel is best just let me knowl